### DISCUSSION TITLE
Mouse moves with focus

### DISCUSSION THREAD

**Discussion by egolep** [qtile/qtile#3053] [Ideas]
At the moment, qtile has a very easy way to let the focus move with the mouse. However, there is no way to do the opposite.
It would be useful to me as it is, but it could also be a way to solve issues like #1419.

There is already a way to do it, even it it means using external tools? If there is, I think it could be useful to write it somewhere explicitly.

**Comment by m-col** [comment]
Sorry, I don't quite understand. What is the opposite of focussing with the mouse?

**Reply by egolep** [reply]
Sorry, I wrote this post two times since I misclicked and moved back on the browser, but reading it now I have to say that this second take is way worse than the first.
The opposite of focusing with the mouse is having the mouse moving with the focus. So if I use the keyboard to change focus, the mouse moves accordingly. Both with changing window (the mouse moves inside the new focused window) or changing screen (the moves moves to the new screen), but the latter is the most important (at least in my opinion)

**Reply by jwijenbergh** [reply]
This is already in qtile with the warp option. Set this to ```True``` in your config https://github.com/qtile/qtile/blob/da93518b2ab924e803552decdb364d64de46088d/libqtile/resources/default_config.py#L168

**Reply by egolep** [reply]
I totally missed it, many thanks!

**Reply by m-col** [reply]
Ah, thanks @jwijenbergh :)
